apps/services/schemas.py

```diff
@@ -13,6 +13,78 @@
+              11: '"Освітні, педагогічні науки"', 52: 'Політологія', 61: 'Журналістика',
+              291:'Міжнародні відносини, суспільні комунікації та регіональні студії', 292: 'Міжнародні економічні відносини',
+              76: '"Підприємництво, торгівля та біржова діяльність"', 241: 'Готельно-ресторанна справа', 242: 'Туризм'}
+
+
+class CreateCustomHostelAccommodationIn(BaseInSchema):
+    rector_first_name: str
+    rector_middle_name: str
+    rector_last_name: str
+    student_first_name: str
+    student_middle_name: str
+    student_last_name: str
+    speciality_code: int
+    speciality_name: str
+    course: int
+    faculty_name: str
+    educ_level: str
+    comment: str
+
+    @root_validator
+    def validate_rector_and_student_names(cls, values):
+        names = ['rector_first_name', 'rector_middle_name', 'rector_last_name', 'student_first_name',
+                 'student_middle_name', 'student_last_name']
+        for name in names:
+            if not values.get(name).istitle():
+                raise ValueError(f"{name.replace('_', ' ').capitalize()} first letter must be uppercase")
+        return values
+
+    @validator('educ_level')
+    def validate_education_level(cls, value):
+        if value not in ['B', 'M']:
+            raise ValueError('Wrong education level value. Should be \'B\' or \'M\', uppercase also')
+        return value
+
+    @validator('faculty_name')
+    def validate_faculty_name(cls, value):
+        if value not in ['Інформаційних технологій', 'Міжнародних відносин і журналістики',
+                         'Міжнародної економіки і підприємництва', 'Фінансів і обліку',
+                         'Менеджмента і маркетингу', 'Економіки і права']:
+            raise ValueError(f'Wrong \'{value}\' number. This faculty doesn\'t exist')
+        return value
+
+    @validator('speciality_code')
+    def validate_speciality_code(cls, value):
+        if value not in speciality.keys():
+            raise ValueError(f'Wrong \'{value}\'number. This speciality doesn\'t exist')
+        return value
+
+    @validator('speciality_name')
+    def validate_speciality_name(cls, value):
+        if value not in speciality.values():
+            raise ValueError(f'Wrong \'{value}\' name. This speciality doesn\'t exist')
+        return value
+
+    @validator('course')
+    def validate_course(cls, value):
+        if value not in [1, 2, 3, 4, 5, 6]:
+            raise ValueError(f"Wrong \'{value}\' number. This course doesn't exist")
+        return value
+
+    @root_validator
+    def validate_speciality_name_and_speciality_code_correspondence(cls, values):
+        if not speciality[values.get('speciality_code')] == values.get('speciality_name'):
+                raise ValueError(f"Speciality code \'{values.get('speciality_code')}\' doesn't correspondence with exists "
+                                 f"\'{values.get('speciality_name')}\' speciality name")
+        return values
+
```

Comment:
1. why didn't u delete old schemas, view, url which was used for creating request?
2. Why didn't u create "Out" schema for a new view?

Reply:
2. Because we decided, that `user_request_id` and `status_id` would be ok. So we have it in CreateRequestOut. If I will add this items in custom new schema - it would be duplicated code

Reply:
ok